hardhat/build/contracts/Identity-address.json

```diff
@@ -0,0 +1 @@
+{"address":"0x57BD87BD93b5377Cf012EC8adFb075E6514256E1"}
```

Comment:
@elblockdude for which network?

Reply:
Hey Darren this will be added at every deploy cz we don't have network section inside the ABI anymore

Reply:
Goes with whatever network is setup on your hardhat config file